package.json

```diff
@@ -21,24 +23,27 @@
   },
   "dependencies": {
     "lodash.camelcase": "^4.3.0",
-    "ramda": "^0.25.0"
+    "ramda": "^0.26.1"
   },
   "devDependencies": {
-    "eslint": "^4.11.0",
-    "eslint-config-airbnb-base": "^12.1.0",
-    "eslint-config-prettier": "^2.7.0",
+    "eslint": "^5.12.0",
+    "eslint-config-airbnb-base": "^13.1.0",
+    "eslint-config-prettier": "^3.3.0",
     "eslint-plugin-import": "^2.8.0",
-    "eslint-plugin-prettier": "^2.3.1",
-    "eslint-watch": "^3.1.3",
-    "husky": "^0.14.3",
-    "jest": "^22.1.3",
-    "lint-staged": "^5.0.0",
+    "eslint-plugin-prettier": "^3.0.1",
+    "eslint-watch": "^4.0.2",
+    "husky": "^1.3.1",
```

Comment:
it doesn't look like you ran the husky upgrade script: https://www.npmjs.com/package/husky#upgrading-from-014

Reply:
ah, thanks! https://github.com/rjhilgefort/export-dir/pull/9